### DISCUSSION TITLE
How could I randomize the texts?

### DISCUSSION THREAD

**Discussion by YaYaYaRong** [dialogic-godot/dialogic#2277] [Q&A] [answered]
I'm trying to let my characters say something different,but I fail to find out the way.What should I do?Save me please......

**Comment by CakeVR** [comment, marked as answer]
Here you go: https://docs.dialogic.pro/writing-text-events.html#4-random-selection

**Reply by YaYaYaRong** [reply]
thanks a lot!!!
